pkg/generators/router/router_test.go

```diff
@@ -319,6 +319,12 @@ func TestGenerate(t *testing.T) {
 					OperationID: "DELETEAnotherOperationID",
 					Group:       "Another",
 				},
+				PUT: &endpoint{
+					Summary:     "PUT Summary",
+					Description: "PUT Description\nbut a multiline this time",
+					OperationID: "PUTAnotherOperationID",
+					Group:       "Another",
+				},
```

Comment:
Please add `\n` to one of descriptions in `"/some/path"`, this endpoint is supposed to check the absence of one of the HTTP methods, that's why `PUT` was not here.

Reply:
the second is also missing a post, but I'll revert this then.

Reply:
changed.